src/Model/Command/UserRecommendation.php

```diff
@@ -227,11 +212,19 @@ protected function setUserId(string $userId): void
         $this->userId = $userId;
     }
 
-    protected function setCount(int $count): void
+    /**
+     * Set number of requested recommendations. The real number of recommended items could be lower or even zero when
+     * there are no items relevant for the user.
+     *
+     * @return $this
+     */
+    public function setCount(int $count): self
     {
         Assertion::greaterThan($count, 0);
 
         $this->count = $count;
+
+        return $this;
     }
 
     protected function setScenario(string $scenario): void
```

Comment:
We tend to order methods by visibility, ie. public > private > protected - so `setUserId()` and `setScenario()` should be after all public methods.

Reply:
fixed